web/.env.defaults

```diff
@@ -5,6 +5,8 @@ WEB_SERVER_PORT=1337
 DOMAIN=lbry.tv
```

Comment:
What is the purpose of having two of these files?

Reply:
The project structure means root for web build is different from root for electron build.

Reply:
can you check in a symlink? maybe a webpack pre-copy?